src/main/scala/io/iohk/ethereum/blockchain/sync/Blacklist.scala

```diff
@@ -0,0 +1,47 @@
+package io.iohk.ethereum.blockchain.sync
+
+import com.github.blemale.scaffeine.{Cache, Scaffeine}
+import io.iohk.ethereum.blockchain.sync.BlacklistSupport.BlackListId
+import io.iohk.ethereum.utils.Logger
+
+import scala.concurrent.duration.FiniteDuration
+
+import scala.jdk.CollectionConverters._
+import scala.jdk.OptionConverters._
+import scala.jdk.DurationConverters._
+
+trait Blacklist {
+  def isBlacklisted(id: BlackListId): Boolean
```

Comment:
nitpick: do you mind refactoring the name to `BlacklistId`?

Reply:
sure, will do :+1: